shakedown/dcos/package.py

```diff
@@ -51,7 +51,8 @@ def install_package(
         options_json=None,
         wait_for_completion=False,
         timeout_sec=600,
-        expected_running_tasks=0
+        expected_running_tasks=0,
+        wait_for_marathon_deployment=True
```

Comment:
This sounds like a larger change to shakedown than I really feel I have the ownership to see through.  

launch_installation() followed by different kinds of wait_for_completion() sounds like a reasonable thing, and a better way to express the complexity that is currently being provided by a long list of arguments to this function.  However, by adding another interface in duplication with the existing one, that feels kind of worse than either unless a plan exists to transition.

In a coding world where I can see all the consumers, I've done this kind of cleanup work several times, but in an world where the consumers are in other products and in other testing infrastructures for other products in repositories I don't' know exist, this doesn't seem achievable in one push.  i also don't feel good about leaving behind a deprecation warning in someone else's codebase that i don't have stewardship for.  

In short I feel doing nothing is better than the feeble degree to which I would travel the suggested path.  I'm okay with doing nothing for now.